Code/Framework/AzFramework/Platform/Mac/AzFramework/Asset/AssetSystemComponentHelper_Mac.cpp

```diff
@@ -41,6 +41,15 @@ namespace AzFramework::AssetSystem::Platform
                 }
             }
 
+             if (!AZ::IO::SystemFile::Exists(assetProcessorPath.c_str()))
```

Comment:
nit: Looks like there's an extra space before the `if` here.

Reply:
Will fix